pkg/sentry/fsimpl/fuse/dev.go

```diff
@@ -328,7 +329,7 @@ func (fd *DeviceFD) EventUnregister(e *waiter.Entry) {
 }
 
 // Seek implements vfs.FileDescriptionImpl.Seek.
-func (fd *DeviceFD) Seek(ctx context.Context, offset int64, whence int32) (int64, error) {
+func (fd *DeviceFD) Seek(_ context.Context, _ int64, _ int32) (int64, error) {
```

Comment:
pls don't do this. There is nothing wrong to have names here and it is more readable.